### PR TITLE
Adding post 2024-02-12 with new Kimai Mobile features

### PR DESCRIPTION
**Pull request by ys-cr**
- Adding post 2024-02-12 with new Kimai Mobile features.
- Adding new URLs to cloudrizon profile.
- Play Store and App Store URLs are replaced with internal link.
- No AI text used.

### CODE REVIEW

File: _posts/2024-02-12-introducing-kimai-mobile-time-tracking-features.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+date: "2024-02-12 12:00:00 +0200"
+author: cloudrizon
+tags: [Apps & Plugins]
+---
+
+## Introducing Kimai Mobile Time Tracking Features 
+
+Kimai Mobile released a couple of new app features that our team is excited to share with you today. 
+
+Kimai Mobile seamlessly integrates with the Kimai Time Tracker Server, providing users with the convenience of managing multiple workspaces concurrently. This allows for efficient time logging across various projects and clients. All time tracking data is securely stored on the self-hosted Kimai Server, ensuring both data privacy and accessibility.
```
Comment (kevinpapst) [inline _posts/2024-02-12-introducing-kimai-mobile-time-tracking-features.md:12]:
```suggestion
is securely stored on the self-hosted Kimai Server 
```
Also when using the SaaS version

File: _posts/2024-02-12-introducing-kimai-mobile-time-tracking-features.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+![Kimai Mobile Download Now]({% link /images/news/Kimai-Mobile-DownloadNow.png %}){:class="image"}
+
+Ready to take control of your time? Start using Kimai Mobile and discover efficient time tracking for yourself. 
+
+### Download Kimai Mobile now:
+
+[Google Play]({% link /_store/cloudrizon-kimai-mobile-app-android.md %})
+[App Store]({% link /_store/cloudrizon-kimai-mobile-app-ios.md %})
+
+For in-depth information about Kimai Mobile and its features, visit: [Kimai Mobile Features](https://www.kimaimobile.com/features/)
```
Comment (kevinpapst) [inline _posts/2024-02-12-introducing-kimai-mobile-time-tracking-features.md:59]: in-depth information belong into the store page, where you can add a link to your own domain, but not here.
remove this line please

**Review by kevinpapst — CHANGES_REQUESTED** [review]
The store pages for Android and iOS have a broken youtube video.
All these informations are not available on the store pages.
I think you should change that, remove the blog post and put that content into the store pages, while fixing the missing videos as well.

File: _posts/2024-02-12-introducing-kimai-mobile-time-tracking-features.md (hunk 1 above)
Comment (ys-cr) [reply _posts/2024-02-12-introducing-kimai-mobile-time-tracking-features.md:12]: Can you clarify this point more? What should be modified?

Comment (kevinpapst) [reply _posts/2024-02-12-introducing-kimai-mobile-time-tracking-features.md:12]: Ups, something went wrong with the suggestion. Updated it.

**Comment by ys-cr** [comment]
Modifications have just been committed and the PR is updated

**Comment by kevinpapst** [comment]
I think you misunderstood me. 
The store pages are literally empty and have no information whatsoever.
Please add content to them instead of the blog.

Don't think about your own SEO when writing that stuff, think about the user.
The Kimai page is not meant to promote your own URLs, it fullfils the task to give the user information and funnel them to ios / android app stores.

Nobody looks into the blog to get these information, people read the store pages:
https://www.kimai.org/store/cloudrizon-kimai-mobile-app-ios.html
https://www.kimai.org/store/cloudrizon-kimai-mobile-app-android.html
And those do not contain any information.

You should incorporate the screenshots into the store pages, add feature lists and then you also add a blog page and link from the blog to the store pages to spread the link love internally.

I have the strong impression that you see kimai.org as some kind of external promotion platform only and this is definitely not what I want.

File: _data/developer.yml [hunk 3]
```diff
@@ -87,6 +87,10 @@ cloudrizon:
     homepage: https://cloudrizon.de/
     username: Cloudrizon
     profile: "#"
+    Community: https://www.kimaimobile.com/the-kimai-mobile-blog
+    Pinterest: https://www.pinterest.de/kimaimobileapp
+    Facebook: https://www.facebook.com/cloudrizon
+    LinkedIn: https://www.linkedin.com/company/cloudrizon
```
[the remark this answers is not in the document]
Comment (kevinpapst) [reply _data/developer.yml]: "remove" does not mean "move to another file"...

**Comment by ys-cr** [comment]
Blog removed. Store pages updated.

**Comment by kevinpapst** [comment]
That looks super, thank you very much! 

Last question: could you please compress the images (at least the mobile break one has 1.4 MB, which is too much).

**Comment by ys-cr** [comment]
Thank you!
My bad, I have just removed the images as we are using more custom images for Android and IOS in the store.

**Review by kevinpapst — APPROVED** [review]

**Comment by kevinpapst** [comment]
Thank you @ys-cr 👍

**Comment by kevinpapst** [comment]
And can you please tell your dev team about these super important upcoming API changes: https://github.com/kimai/kimai/pull/4637

**Pull request merged**